docs/build/Build-for-OpenWrt.md

```diff
@@ -0,0 +1,99 @@
+# IoT.js for OpenWrt build guide
+
+This document describes the steps required to compile the IoT.js
+for OpenWrt. For target device the TP-Link WR1043ND v1.x router is
+used. Please be advised, that if you have a different one minor
```

Comment:
Suggestion for readability :
Please be advised, that if you have a different one, minor modifications to this document could be required